src/main/java/baseball/User.java

```diff
@@ -0,0 +1,50 @@
+package baseball;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.HashSet;
+import java.util.Set;
+
+public class User {
+    private static final int INPUT_LENGTH = 3;
+    private static final String INPUT_MESSAGE = "숫자를 입력해주세요 : ";
+    private static final String INPUT_REGEX = "^[1-9]{3}$";
+
+    private static final String INPUT_NOT_MATCH_ERROR_MESSAGE = "입력값이 잘못되었습니다.";
+    private static final String INPUT_DUPLICATE_ERROR_MESSAGE = "중복된 숫자가 있습니다.";
+
+    int[] userBaseBallNumber = new int[INPUT_LENGTH];
```

Comment:
캡슐화를 위해 private으로 변경!!
```suggestion
    private int[] userBaseBallNumber = new int[INPUT_LENGTH];
```